eras/shelley/impl/src/Cardano/Ledger/Shelley/Scripts.hs

```diff
@@ -197,25 +230,26 @@ eqMultiSigRaw t1 t2 = go (getMemoRawType t1) (getMemoRawType t2)
 -- | Script evaluator for native multi-signature scheme. 'vhks' is the set of
 -- key hashes that signed the transaction to be validated.
 evalMultiSig ::
-  Era era =>
+  ShelleyEraScript era =>
```

Comment:
If you don't do this restriction then all of the sudden the `"Impossible"` case will be possible for timelocks
```suggestion
  (ShelleyEraScript era, NativeScript era ~ MultiSig era) =>
```

Reply:
Makes sense, thank you